test_scripts/API/AddSubMenu/ATF_AddSubMenu_Negative_SubMenuIcon_OutUpperBound.lua

```diff
@@ -54,7 +54,7 @@ function Test:Precondition_ActivateApp()
   end)
   EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN", audioStreamingState = "AUDIBLE"})
 end
-commonSteps:PutFile("PutFile_strOutofBoundFileName", strOutofBoundFileName)
+-- commonSteps:PutFile("PutFile_strOutofBoundFileName", strOutofBoundFileName)
```

Comment:
@dboltovskyi, if the script will not use PutFile() as precondition, please delete this line.